package-lock.json

```diff
@@ -2488,12 +2488,6 @@
 				"node": "^10 || ^12 || >=14"
 			}
 		},
-		"node_modules/autoprefixer/node_modules/caniuse-lite": {
-			"version": "1.0.30001171",
-			"resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001171.tgz",
-			"integrity": "sha512-5Alrh8TTYPG9IH4UkRqEBZoEToWRLvPbSQokvzSz0lii8/FOWKG4keO1HoYfPWs8IF/NH/dyNPg1cmJGvV3Zlg==",
-			"dev": true
-		},
```

Comment:
Also bumped browserslist while I was at it, award to the noisiest warning in the world goes to...